bumpCversion.py

```diff
@@ -133,17 +169,36 @@ def main():
     # Parse command line arguments
     args = parse_args()
 
-    # users desired part (major, minor, or patch) to bump
+    # Users desired part (major, minor, or patch) to bump
     part_to_bump = args.part
 
-    # get file we're interested in
+    # Get file we're interested in
     target_files = get_target_files(args)
 
+    # Create list of found versions (length should be == 1)
+    versions = set()
+
+    # Check for dry run
+    if args.dry_run:
+        print_dry(args, target_files.copy(), versions)
+        exit()
+
+    # Check to see if version number is the same across all files
+    print("Checking component [" + str(args.component) +
+          "] for verison number congruence...\n")
+    valid_version_congruence(args, target_files.copy(), versions)
+
     while target_files:
 
+        print("Checking component [" +
+              str(args.component) + "] for file type...")
+
         # Creates object representing first file from `target_files`, then pops it off list
         filetype = get_filetype_object(args, target_files)
 
+        # Print class type (named after filetype)
+        print("Using " + type(filetype).__name__ + " class...")
```

Comment:
I'm not sure that this information is useful to print out to the user. I think printing out the "Target file:" probably tells the user enough. What do you think?

Reply:
Agreed